frontend/packages/core/src/AppProvider/themes.tsx

```diff
@@ -1,9 +1,18 @@
 import React from "react";
 import { ThemeProvider as EmotionThemeProvider } from "@emotion/react";
-import { createMuiTheme, CssBaseline, MuiThemeProvider, ThemeOptions } from "@material-ui/core";
-import { useTheme as useMuiTheme } from "@material-ui/core/styles";
-import type { PaletteOptions } from "@material-ui/core/styles/createPalette";
-import { StylesProvider } from "@material-ui/styles";
+import type { Theme as MuiTheme } from "@mui/material";
+import {
+  CssBaseline,
+  DeprecatedThemeOptions,
+  StyledEngineProvider,
+  ThemeProvider,
+} from "@mui/material";
+import { createTheme, PaletteOptions, useTheme as useMuiTheme } from "@mui/material/styles";
+import { StylesProvider } from "@mui/styles";
+
+declare module "@mui/styles/defaultTheme" {
+  interface DefaultTheme extends MuiTheme {}
+}
```

Comment:
locally these lines are throwing errors for me - `Cannot find module` & `Invalid module name in augmentation`. Curious if you're seeing this as well or if it's a local issue

Reply:
Can't say I've seen those issues, are you building this branch or the big one?

Reply:
oh yeah this was when I was just trying to build this branch, building the big branch works for me.